PlaywrightSolar/PageObjects/InvertersPage.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightSolar.PageObjects;
+
+public class InvertersPage(IPage page)
+{
+    private const string InvertersPageUrl = "https://solartechnology.com.ua/shop/inverters";
+
+    // Methods
+    public async Task GoToInvertersPage()
+    {
+        await page.GotoAsync(InvertersPageUrl);
+        Assert.That(page.Url, Is.EqualTo(InvertersPageUrl), "Failed to navigate to Inverters page");
+    }
+
+    public async Task GetProductFromList()
+    {
+        var listOfProductsOnPage = await page.QuerySelectorAllAsync(".prod-holder");
+        var selectedProduct = listOfProductsOnPage[0];
```

Comment:
Missed the same comment here. Magic number 0 and method name should be AddProductToCart(int productIndex)